<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlShortcutDialog</class>
 <widget class="QDialog" name="SqlShortcutDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotkey filters dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/svg/grey/database.svg</normaloff>:/svg/grey/database.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="testGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Test query results table</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="hideTestBoxButton">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>35</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Hide</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="testResultTable">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="shortcutIdEdit">
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="shortcutNameEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="shortcutQueryEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Sql query</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testSqlQueryButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Test   </string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="shortcutsTable">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="addSqlShortcutButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editSqlShortcutButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Edit  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeSqlShortcutButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>35</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>     Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SqlShortcutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideTestBoxButton</sender>
   <signal>clicked()</signal>
   <receiver>SqlShortcutDialog</receiver>
   <slot>hideTestsTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testSqlQueryButton</sender>
   <signal>clicked()</signal>
   <receiver>SqlShortcutDialog</receiver>
   <slot>testButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addSqlShortcutButton</sender>
   <signal>clicked()</signal>
   <receiver>SqlShortcutDialog</receiver>
   <slot>saveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSqlShortcutButton</sender>
   <signal>clicked()</signal>
   <receiver>SqlShortcutDialog</receiver>
   <slot>editButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>813</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSqlShortcutButton</sender>
   <signal>clicked()</signal>
   <receiver>SqlShortcutDialog</receiver>
   <slot>removeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hideTestsTable()</slot>
  <slot>showTestsTable()</slot>
  <slot>testButtonClicked()</slot>
  <slot>saveButtonClicked()</slot>
  <slot>editButtonClicked()</slot>
  <slot>removeButtonClicked()</slot>
 </slots>
</ui>
